frontend/src/Components/Markdown/InlineMarkdown.js

```diff
@@ -33,13 +34,28 @@ class InlineMarkdown extends Component {
       }
     }
 
+    // replace blocks in the string surrounded in backticks with <code>
+    // if the first character is a backtick then we ignore the first split and start directly with a code block
+    if (code) {
+      const codeSplit = code.split('`');
```

Comment:
Instead of splitting can we use regex and loop through the matches like we do with the links above?

Reply:
I've revised this to a regex matching.